From a176cf1bc32727b438b2c1d9f6f1e4cb868c2f6f Mon Sep 17 00:00:00 2001
From: Simon Kelley <simon@thekelleys.org.uk>
Date: Fri, 14 Oct 2022 11:46:13 +0100
Subject: [PATCH 26/31] Move fast-dns-retry and use-stale-cache writeups in the
 CHANGELOG.

These are 2.88 changes, but the branch merge put them unde 2.87.
---
 CHANGELOG | 32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/CHANGELOG b/CHANGELOG
index db13583..1d2391e 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -2,7 +2,23 @@ version 2.88
 	Fix bug in --dynamic-host when an interface has /16 IPv4
 	address. Thanks to Mark Dietzer for spotting this.
 	
+	Add --fast-dns-retry option. This gives dnsmasq the ability
+	to originate retries for upstream DNS queries itself, rather
+	than relying on the downstream client. This is most useful
+	when doing DNSSEC over unreliable upstream network. It comes
+	with some cost in memory usage and network bandwidth.
+
+	Add -use-stale-cache option. When set, if a DNS name exists
+	in the cache, but its time-to-live has expired, dnsmasq will
+	return the data anyway. (It attempts to refresh the
+	data with an upstream query after returning the stale data.)
+	This can improve speed and reliability. It comes 
+	at the expense of sometimes returning out-of-date data and
+	less efficient cache utilisation, since old data cannot be
+	flushed when its TTL expires, so the cache becomes
+	strictly least-recently-used.
 
+	
 version 2.87
         Allow arbitrary prefix lengths in --rev-server and
 	--domain=....,local
@@ -89,23 +105,7 @@ version 2.87
 	client facing network. Thanks to Luis Thomas for spotting this
 	and initial patch.
 
-	Add --fast-dns-retry option. This gives dnsmasq the ability
-	to originate retries for upstream DNS queries itself, rather
-	than relying on the downstream client. This is most useful
-	when doing DNSSEC over unreliable upstream network. It comes
-	with some cost in memory usage and network bandwidth.
-
-	Add -use-stale-cache option. When set, if a DNS name exists
-	in the cache, but its time-to-live has expired, dnsmasq will
-	return the data anyway. (It attempts to refresh the
-	data with an upstream query after returning the stale data.)
-	This can improve speed and reliability. It comes 
-	at the expense of sometimes returning out-of-date data and
-	less efficient cache utilisation, since old data cannot be
-	flushed when its TTL expires, so the cache becomes
-	strictly least-recently-used.
 
-	
 version 2.86
 	Handle DHCPREBIND requests in the DHCPv6 server code.
 	Thanks to Aichun Li for spotting this omission, and the initial
-- 
2.25.1

